### DISCUSSION TITLE
12月10日固件，手机看视频会断网

### DISCUSSION THREAD

**Discussion by ThermalEng** [Boos4721/openwrt#49] [Bugs/问题反馈]
Sat Dec 11 00:17:32 2021 daemon.info hostapd: wlan1: STA 98:e0:d9:cf:0b:0e IEEE 802.11: authenticated
Sat Dec 11 00:17:32 2021 daemon.info hostapd: wlan1: STA 98:e0:d9:cf:0b:0e IEEE 802.11: associated (aid 5)
Sat Dec 11 00:17:32 2021 daemon.notice hostapd: wlan1: AP-STA-CONNECTED 98:e0:d9:cf:0b:0e
Sat Dec 11 00:17:32 2021 daemon.info hostapd: wlan1: STA 98:e0:d9:cf:0b:0e RADIUS: starting accounting session D5C6615122C73F7B
Sat Dec 11 00:17:32 2021 daemon.info hostapd: wlan1: STA 98:e0:d9:cf:0b:0e WPA: pairwise key handshake completed (RSN)
Sat Dec 11 00:17:32 2021 daemon.notice hostapd: wlan1: EAPOL-4WAY-HS-COMPLETED 98:e0:d9:cf:0b:0e
Sat Dec 11 00:17:43 2021 daemon.notice hostapd: wlan1: AP-STA-DISCONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:17:44 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: authenticated
Sat Dec 11 00:17:44 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: associated (aid 3)
Sat Dec 11 00:17:44 2021 daemon.notice hostapd: wlan1: AP-STA-CONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:17:44 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac RADIUS: starting accounting session 46A4FD9C71898122
Sat Dec 11 00:17:44 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac WPA: pairwise key handshake completed (RSN)
Sat Dec 11 00:17:44 2021 daemon.notice hostapd: wlan1: EAPOL-4WAY-HS-COMPLETED fe:aa:c3:af:d5:ac
Sat Dec 11 00:18:17 2021 daemon.notice hostapd: wlan1: AP-STA-DISCONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:18:19 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: authenticated
Sat Dec 11 00:18:19 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: associated (aid 3)
Sat Dec 11 00:18:19 2021 daemon.notice hostapd: wlan1: AP-STA-CONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:18:19 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac RADIUS: starting accounting session 0815091DA90FCFBE
Sat Dec 11 00:18:19 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac WPA: pairwise key handshake completed (RSN)
Sat Dec 11 00:18:19 2021 daemon.notice hostapd: wlan1: EAPOL-4WAY-HS-COMPLETED fe:aa:c3:af:d5:ac
Sat Dec 11 00:18:41 2021 daemon.notice hostapd: wlan1: AP-STA-DISCONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:18:42 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: authenticated
Sat Dec 11 00:18:42 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: associated (aid 3)
Sat Dec 11 00:18:42 2021 daemon.notice hostapd: wlan1: AP-STA-CONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:18:42 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac RADIUS: starting accounting session C48D0FE30D3D5040
Sat Dec 11 00:18:42 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac WPA: pairwise key handshake completed (RSN)
Sat Dec 11 00:18:42 2021 daemon.notice hostapd: wlan1: EAPOL-4WAY-HS-COMPLETED fe:aa:c3:af:d5:ac
Sat Dec 11 00:19:09 2021 daemon.notice hostapd: wlan1: AP-STA-DISCONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:19:11 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: authenticated
Sat Dec 11 00:19:11 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: associated (aid 3)
Sat Dec 11 00:19:11 2021 daemon.notice hostapd: wlan1: AP-STA-CONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:19:11 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac RADIUS: starting accounting session 494C286415EF90EC
Sat Dec 11 00:19:11 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac WPA: pairwise key handshake completed (RSN)
Sat Dec 11 00:19:11 2021 daemon.notice hostapd: wlan1: EAPOL-4WAY-HS-COMPLETED fe:aa:c3:af:d5:ac
Sat Dec 11 00:19:39 2021 daemon.notice hostapd: wlan1: AP-STA-DISCONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:19:41 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: authenticated
Sat Dec 11 00:19:41 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: associated (aid 3)
Sat Dec 11 00:19:41 2021 daemon.notice hostapd: wlan1: AP-STA-CONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:19:41 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac RADIUS: starting accounting session 427F1FA4D2A0BB6D
Sat Dec 11 00:19:41 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac WPA: pairwise key handshake completed (RSN)
Sat Dec 11 00:19:41 2021 daemon.notice hostapd: wlan1: EAPOL-4WAY-HS-COMPLETED fe:aa:c3:af:d5:ac
Sat Dec 11 00:19:55 2021 daemon.notice hostapd: wlan1: AP-STA-DISCONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:19:57 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: authenticated
Sat Dec 11 00:19:57 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: associated (aid 3)
Sat Dec 11 00:19:57 2021 daemon.notice hostapd: wlan1: AP-STA-CONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:19:57 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac RADIUS: starting accounting session 6BFFE81F9558A380
Sat Dec 11 00:19:57 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac WPA: pairwise key handshake completed (RSN)
Sat Dec 11 00:19:57 2021 daemon.notice hostapd: wlan1: EAPOL-4WAY-HS-COMPLETED fe:aa:c3:af:d5:ac
Sat Dec 11 00:20:00 2021 cron.err crond[2686]: USER root pid 21228 cmd /bin/sh /root/my_pppoe.sh runA >> /var/log/my_pppoe.log 2>&1
Sat Dec 11 00:20:02 2021 daemon.notice hostapd: wlan1: AP-STA-DISCONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:20:04 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: authenticated
Sat Dec 11 00:20:04 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: associated (aid 3)
Sat Dec 11 00:20:04 2021 daemon.notice hostapd: wlan1: AP-STA-CONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:20:04 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac RADIUS: starting accounting session 3D045AEFC9EEB1C8
Sat Dec 11 00:20:04 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac WPA: pairwise key handshake completed (RSN)
Sat Dec 11 00:20:04 2021 daemon.notice hostapd: wlan1: EAPOL-4WAY-HS-COMPLETED fe:aa:c3:af:d5:ac
Sat Dec 11 00:20:09 2021 daemon.notice hostapd: wlan1: AP-STA-DISCONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:20:11 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: authenticated
Sat Dec 11 00:20:11 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac IEEE 802.11: associated (aid 3)
Sat Dec 11 00:20:11 2021 daemon.notice hostapd: wlan1: AP-STA-CONNECTED fe:aa:c3:af:d5:ac
Sat Dec 11 00:20:11 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac RADIUS: starting accounting session 9EFAD95DAC470138
Sat Dec 11 00:20:11 2021 daemon.info hostapd: wlan1: STA fe:aa:c3:af:d5:ac WPA: pairwise key handshake completed (RSN)
Sat Dec 11 00:20:11 2021 daemon.notice hostapd: wlan1: EAPOL-4WAY-HS-COMPLETED fe:aa:c3:af:d5:ac
Sat Dec 11 00:20:18 2021 authpriv.info dropbear[14603]: Exit (root) from <192.168.50.31:62256>: Disconnect received

**Comment by Boos4721** [comment]
` USER root pid 21228 cmd /bin/sh /root/my_pppoe.sh runA >> /var/log/my_pppoe.log 2>&1`
ty drop it, and clean your router config.

**Reply by ThermalEng** [reply]
路由器5g频段的接收协商速率掉的很快，稍微远一点就只有20mhz，48mbs了。设置信道161，80mhz，老固件运行正常。

**Comment by ThermalEng** [comment]
wifi5的设备都正常  电脑wifi6速度之前100m的地方现在只有10m 手机不停重连。pppoe那个cron停掉也是一样

**Comment by ThermalEng** [comment]
定位问题了。问题应该在上传，在信号稍弱的情况下，手机无论是5g还是2.4g都无法上传，导致连接断开。

**Comment by ThermalEng** [comment]
修改ax为ac后2.4g和5g均正常
